### PR TITLE
Fixes support for uninstalling the eBPF collector in the uninstaller and fixes a minor bug

### LINKED ISSUE #8189: Add uninstaller support for eBPF kernel package and shared libraries
##### Feature idea summary

As eBPF is a "tech preview" feature we release for the recent `v1.20.0` we did not have time to add uninstaller support for what gets installed on a user's system for the eBPF shared libraries. This issue is to track the implementation of this and should be added soon. So far we have not changed how this gets installed so the "undo" process should be fairly simple.

##### Expected behavior

Running the NetData uninstaller also removes the installed eBPF shared libraries.

### LINKED ISSUE #8428: Installer stop_all_netdata uv_pipe_connect() err
#### Question summary

In Ubuntu 18.04 I get the following installer error:

```stop_all_netdata uv_pipe_connect() : no such file or directory```

I have my netdata running as user `chris`, but I don't know if it's related.
Do we know where it comes from?

### PR DESCRIPTION
**Pull request by prologic**
##### Summary

As per title

- Fixes #8428
- Fixes #8189

##### Component Name

- area/packaging

##### Test Plan

#8428:

- [x] Install netdata on a fresh CentOS 7 system from kickstart
- [x] Re-install netdata over the top and observe the error `stop_all_netdata uv_pipe_connect(): no such file or directory`

#8189:

- [x] Install netdata on a fresh Debian 10 (buster) system with `--enable-epbf`
- [x] Run the uninstaller and observe that all eBPF related files are removed.

##### Additional Information

### CODE REVIEW

**Comment by prologic** [comment]
Before:

```sh
Stopping all netdata threads
[/home/prologic/netdata]# stop_all_netdata uv_pipe_connect(): no such file or directory
Make sure the netdata service is running.
 OK
```

After:

```sh
Stopping all netdata threads
[/home/prologic/netdata]# stop_all_netdata  OK

Starting netdata using command 'systemctl start netdata'
[/home/prologic/netdata]# systemctl start netdata  OK

 OK  netdata started!
```

**Comment by prologic** [comment]
Before:

```sh
root@debian:~# /usr/libexec/netdata/netdata-uninstaller.sh -f -y
.
.
.
root@debian:~# find / | grep 'bpf_kernel'
/usr/local/lib/libbpf_kernel.so
/usr/local/lib/libbpf_kernel.so.0
```

After:

```sh
root@debian:~# /usr/libexec/netdata/netdata-uninstaller.sh -f -y
.
.
.
 Finding lib directory ...
[/root]# ldconfig \n OK  ldconfig

ePBF uninstall all done!
.
.
.
root@debian:~# find / | grep 'bpf_kernel'
root@debian:~#
```

File: packaging/installer/netdata-uninstaller.sh [hunk 1]
```diff
@@ -24,422 +24,450 @@ ENVIRONMENT_FILE="/etc/netdata/.environment"
...
+  libdir=
+  libdir="$(ldconfig -v 2> /dev/null | grep ':$' | sed -e 's/://' | sort -r | grep 'usr' | head -n 1)"
+  if [ -z "${libdir}" ]; then
+    libdir="$(ldconfig -v 2> /dev/null | grep ':$' | sed -e 's/://' | sort -r | head -n 1)"
+  fi
+
+  if [ -z "${libdir}" ]; then
+    echo >&2 "Unable to find the system lib directory we installed eBPF ebpf_kernel.so to ..."
+    user_input "Press ENTER to search your system (which may take some time) ..."
+    find / -type f -name 'bpf_kernel.so' -o -name 'bpf_kernel.so.o'
```
Comment (thiagoftsm) [inline packaging/installer/netdata-uninstaller.sh]: I understand that find can search for partial names, but should not we use the complete name( `libbpf_kernel.so` instead `bpf_kernel.so.`?
Another thing, please, change `bpf_kernel.so.o` to `bpf_kernel.so.0`, the second name is wrong.

Comment (prologic) [reply packaging/installer/netdata-uninstaller.sh]: Thanks Fixed :)

**Pull request merged**